### PR TITLE
Feature map diplay

### PR DESCRIPTION
**Pull request by teasmade**
Base map component in place with bike station markers + popups, basic geolocalisation functionality in place.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import SampleComponent from './components/SampleComponent';
+import Map from './components/Map';
+// import SampleComponent from './components/SampleComponent';
```
Comment (bastienapp) [inline src/App.jsx]: Pas de code mort, tu peux supprimer le SampleComponent  plutôt que le commenter :)

File: src/App.jsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 
 function App() {
   return (
     <div className="App">
-      <SampleComponent />
+      {/* <SampleComponent /> */}
```
Comment (bastienapp) [inline src/App.jsx]: ligne à supprimer

**Review by bastienapp — CHANGES_REQUESTED** [review]
deux petits changements :)

**Comment by teasmade** [comment]
Requested changes done, do we need to make a new pull request?

File: src/App.jsx (hunk 1 above)
Comment (teasmade) [reply src/App.jsx]: Change done

File: src/App.jsx (hunk 2 above)
Comment (teasmade) [reply src/App.jsx]: Change done

**Review by bastienapp — APPROVED** [review]
👍

**Pull request merged**